### ISSUE TITLE
Issue with cross_validation

### ISSUE THREAD

**Issue by cereusperuv** [facebook/prophet#1825]
Hi!

First, thanks for a great forecasting package! I have used it alot.

I am having issues using the cross validation functionality. The function `fbprophet.diagnostics.cross_validation` does not accept the argument `disable_tqdm`, as it should per the v 0.7.1 GitHub source code. Please see code and output below.

Code
```
import fbprophet

print(f'fbprophet version:  {fbprophet.__version__}\n')

m = fbprophet.Prophet(
    yearly_seasonality=10,
    weekly_seasonality=True,
    daily_seasonality=False)
m.fit(df)
cv_df = fbprophet.diagnostics.cross_validation(
    m, cutoffs=start_dates, horizon='31 days', disable_tqdm=False)
```

Ouput
```
fbprophet version:  0.7.1

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-26-9c50f9eae5ec> in <module>
      8     daily_seasonality=False)
      9 m.fit(df)
---> 10 cv_df = fbprophet.diagnostics.cross_validation(
     11     m, cutoffs=start_dates, horizon='31 days', disable_tqdm=False)

TypeError: cross_validation() got an unexpected keyword argument 'disable_tqdm'
```

I am running this within a Jupyter notebook in a Docker container based on the `python:3.8-slim` image. Prophet works well except for this error. Note that when calling the function without the argument `disable_tqdm`, the error code contains the following, among other things, which also does not match the GitHub repo code.

```
/usr/local/lib/python3.8/site-packages/fbprophet/diagnostics.py in cross_validation(model, horizon, period, initial, parallel, cutoffs)
    187         predicts = [
    188             single_cutoff_forecast(df, model, cutoff, horizon, predict_columns)
--> 189             for cutoff in tqdm(cutoffs)
    190         ]
    191 
```

**Comment by tcuongd** [comment]
0.7.1 was released on Sep 6 2020 (https://pypi.org/project/fbprophet/), but the PR that added the `disable_tqdm` flag was only added in Nov 7 2020 (https://github.com/facebook/prophet/pull/1706), so we'll need to wait for the next release for that feature.

That said, you should still be able to run `cross_validation()` without that flag. Could you please post the full error message?

**Comment by cereusperuv** [comment]
Aha, I see. Thanks for the reply! My first solution was to write my own `cross_validation` function.

Without the flag it failed for different reasons connected to the progress bar, that were resolved when I updated `ipywidgets` to v. 7.4.2 in my container (I have no idea why though, but that solution was the first I found online). I have pasted the final part of the error message below. The rest of it got lost after I updated `ipywidgets`. If its important for the development of this project I can rebuild my Docker image with the old `ipywidgets` version to re-generate it in its entirety.

```
ImportError: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
```
